modules/sdk-coin-eos/src/eos.ts

```diff
@@ -360,11 +360,11 @@ export class Eos extends BaseCoin {
    * Convert a currency amount represented in base units (satoshi, wei, atoms, drops, stroops)
    * to big units (btc, eth, xrp, xlm)
    */
-  baseUnitsToBigUnits(baseUnits: string | number): string {
-    const dividend = this.getBaseFactor();
+  baseUnitsToBigUnits(baseUnits: string | number, chex?: boolean): string {
+    const dividend = chex ? 1e8 : this.getBaseFactor();
```

Comment:
`this.getBaseFactor()` should be left as is. we have an `EosToken` subclass that will populate this with the correct value.